morello/system_config/cpu.py

```diff
@@ -242,4 +255,8 @@ def _parse_benchmark_output(output: str) -> float:
 def _clang_path() -> Path:
     if "CLANG" in os.environ:
         return Path(os.environ["CLANG"])
+    elif platform.system() == "Darwin":
```

Comment:
@samkaufman Thank you for your review! May I kindly ask what led you to prefer that option? In my opinion, it would be helpful if Morello would indicate that CLANG is not set from the start of a run.

Reply:
We want a good error message if CLANG isn't set, but let's not put ourselves on the hook to hardcode default paths for every platform.